src/filesys.h

```diff
@@ -69,8 +69,8 @@ std::string TempPath();
 
 /* Multiplatform */
 
-// The path itself not included
-void GetRecursiveSubPaths(const std::string &path, std::vector<std::string> &dst);
+// The path itself not included, returns a list of all subpaths.
+void GetRecursiveSubPaths(const std::string &path, std::vector<std::string> &dst, bool list_files, std::vector<char> ignore);
```

Comment:
This line is way too long.
There is no description whatsoever about what these new parameters do.
ignore is supposed to be a const reference if you don't plan on modifying it, or perhaps a const pointer if you want to make it an optional parameter.